src/core/algorithms/md/hymd/lattice_traverser.cpp

```diff
@@ -6,12 +6,16 @@ namespace algos::hymd {
 
 bool LatticeTraverser::TraverseLattice(bool const traverse_all) {
     using model::Index;
+    LatticeStatistics lattice_statistics;
     while (level_getter_->AreLevelsLeft()) {
```

Comment:
```suggestion
    while (level_getter_->AreLevelsLeft()) {
        LatticeStatistics lattice_statistics;
```
Maybe your version is also okay, but this is what I'm looking for.